unidist/core/backends/mpi/core/controller/object_store.py

```diff
@@ -33,6 +34,10 @@ def __init__(self):
         self._data_id_counter = 0
         # Data serialized cache
         self._serialization_cache = {}
+        # Data sizes
+        self._data_sizes = defaultdict(int)
+        # Mapping from python identity id() to DataID {id() : DataID}
+        self._identity_data_id_map = defaultdict(lambda: None)
```

Comment:
Why do we need this map?

Reply:
This is no longer required, will remove this.

The original purpose of this map was to make sure to not send same python object to workers multiple times, as put was planned to be used in submit method for all arguments but this was later changed.